apis/scheduling/v1alpha1/types.go

```diff
@@ -47,13 +47,15 @@ type ElasticQuota struct {
 // ElasticQuotaSpec defines the Min and Max for Quota.
 type ElasticQuotaSpec struct {
 	// Min is the set of desired guaranteed limits for each named resource.
+	// +kubebuilder:default={cpu:0,memory:0,ephemeral-storage:0}
 	// +optional
-	Min v1.ResourceList `json:"min,omitempty" protobuf:"bytes,1,rep,name=min, casttype=ResourceList,castkey=ResourceName"`
+	Min v1.ResourceList `json:"min" protobuf:"bytes,1,rep,name=min, casttype=ResourceList,castkey=ResourceName"`
 
 	// Max is the set of desired max limits for each named resource. The usage of max is based on the resource configurations of
 	// successfully scheduled pods.
+	// +kubebuilder:default={cpu:9223372036854775807,memory:9223372036854775807,ephemeral-storage:9223372036854775807}
```

Comment:
this looks a bit strange to me. Is there a better way to represent this?

Is it worth introducing `map[ResourceName]*resource.Quantity`?

Reply:
I understand from your comment that you suggested creating a variable and assigning it to `kubebuilder`'s marker. I tried it, but I couldn't assign a variable to `kubebuilder`'s marker. I also read `kubebuilder`'s documentation, but I couldn't find a good way to do it.
If there is a better way, please let me know. Also, please let me know if I misunderstood your comment.

This is what I did.
First, I created a variable
`MaxBound := "{cpu:9223372036854775807,memory:9223372036854775807,ephemeral-storage:9223372036854775807}"` and used it in `kubebuilder`'s marker. However it doesn't work.

Reply:
It's also a little strange to me.  Does it look like this if I describe the CR?
```yaml
  max:
    cpu: 9223372036854775807
    ephemeral-storage: 9223372036854775807
    memory: 9223372036854775807
  min:
    cpu: 10
    ephemeral-storage: 0
    memory: 0
```

I prefer not to set the default value for min and max. We can set the default value in the scheduler when we init the cache but do not need to put it in CR.

Reply:
Thank you for your comments. It is very helpful for me.

If you apply the following resource, yes you can get CR as you wrote.
```
  min:
    cpu: 10
    ephemeral-storage: 0
    memory: 0
```

I see. Then I will handle it in the cache and try not to change default CR.